src/main/java/ch/ivyteam/ivy/maven/StartTestEngineMojo.java

```diff
@@ -116,7 +116,7 @@ private File copyEngineToTarget(File cachedEngineDir) throws MojoExecutionExcept
     }
     catch (IOException ex)
     {
-      getLog().warn("Could not copy engine from: " + cachedEngineDir + " to " + targetEngine);
+      getLog().warn("Could not copy engine from: " + cachedEngineDir + " to " + targetEngine + ex);
```

Comment:
you should log the stacktrace                instad of `+ ex` use` , ex`

Reply:
note the "Outdated" label

after committing the "+ ex" I immediately realized my mistake and changed it to ", ex" but thank you anyway :+1: 

master: https://github.com/axonivy/project-build-plugin/blob/master/src/main/java/ch/ivyteam/ivy/maven/StartTestEngineMojo.java